runtime/doc/term.txt

```diff
@@ -35,6 +48,89 @@ can do this best in your vimrc.  Example: >
      ... vt100, vt102 maps and settings ...
    endif
 <
+					*scroll-region* *xterm-scroll-region*
+Where possible, Nvim will use the terminal's ability to set a scroll region in
+order to redraw faster when a window is scrolled.  If the terminal's terminfo
+description describes an ability to set top and bottom scroll margins, that is
+used.
+
+This will not speed up scrolling in a window that is not the full width of the
+terminal.  Xterm has an extra ability, not described by terminfo, to set left
+and right scroll margins as well.  If Nvim detects that the terminal is Xterm,
+it will make use of this ability to speed up scrolling that is not the full
```

Comment:
I think that it's not really necessary to _completely_ repeat what was said only moments before.  (-: